compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/DefaultViewConfiguration.skiko.kt

```diff
@@ -19,7 +19,7 @@ package androidx.compose.ui.platform
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.Density
 
-class DefaultViewConfiguration(private val density: Density) : ViewConfiguration {
+class DefaultViewConfiguration(private val densityProvider: () -> Density) : ViewConfiguration {
```

Comment:
I remember we had discussed this briefly last week.

Could you please remind me why it needs to be a density provider instead of an explicit density? 

The concern is that this class is public and this change is a breaking change.

Reply:
Yeah. It's public... I will revert this change.
I found, that change of density not recreate DefaultViewConfiguration instance